### ISSUE TITLE
Github subdomains are yellowlisted on github

### ISSUE THREAD

**Issue by pde** [EFForg/privacybadgerfirefox#648]
I see a bunch of github subdomains (api.github.com, live.github.com, asses-cdn.github.com) yellowlisted on github.com.  In practice this has small consequences, such as breaking rollover information in the contribution history on user profile pages.  We should check that this is the intended behaviour in the design...

**Comment by pde** [comment]
Currently unclear if this is really a site bug, a failure to match the design, or simply a consequence of the design.

**Comment by rugk** [comment]
What I saw was that especially the avatar subdomains are **not** yellowlisted (e.g. avatars0.githubapp.com, avatars3.githubapp.com). This - obviously - causes display problems with some avatars.

**Comment by cooperq** [comment]
This is definitely not a thing that should ever happen. subdomains of github.com shouldn't even show up in privacy badger on github.com 
@pde lets try to reproduce this bug together.

**Comment by cooperq** [comment]
fwiw I am unable to reproduce this in firefox

**Comment by rugk** [comment]
Interesting. Did not know that this is so unreproducible, because it happens all the time in my FF. (v41)
![github](https://cloud.githubusercontent.com/assets/11966684/10923682/b50ff4b4-8282-11e5-9cff-2a89c11bf5a0.png)

**Comment by cooperq** [comment]
damn! @rugk can you also confirm that you have the latest version of privacy badger (1.0.3) and please tell me what other privacy related settings you have turned on (cookies, no history, etc.)  thank you!

**Comment by cooperq** [comment]
looks like this is the same bug as #118 this is a top priority for me.

**Comment by rugk** [comment]
> can you also confirm that you have the latest version of privacy badger (1.0.3) 
Yes, it shows !1.0.3" in about:addons.

> please tell me what other privacy related settings you have turned on (cookies, no history, etc.) 

The only special I can see is "[blocking third-party cookies](https://support.mozilla.org/en-US/kb/disable-third-party-cookies)". Except of this, Firefox [internal tracking protection](https://support.mozilla.org/en-US/kb/tracking-protection-pbm) (also turned on for browsing in non-private windows), DNT and other add-ons which may also block some assets I have nothing else.

> looks like this is the same bug as #118 this is a top priority for me. 

I have no issues with cookies any logins, but I did not tested it on the sites mentioned in the issues (I also do not have accounts there).

**Comment by jawz101** [comment]
![mine looks like this](https://cloud.githubusercontent.com/assets/14151703/11075643/e49def68-87bb-11e5-8e79-52a0c2dbffa2.png)

**Comment by cooperq** [comment]
oh interesting, I wonder if SDC is the problem here. @rugk do you also have self destructing cookies extension?  Can you both list the extensions you have installed? (and you @pde if you manage to reproduce this).

**Comment by rugk** [comment]
>  @rugk do you also have self destructing cookies extension?

Nope. Only Firefox setting for not accepting third-party cookies.
And BTW I do not see any google domains blocked here at GitHub.
![pbgithubsubdomains](https://cloud.githubusercontent.com/assets/11966684/11076753/cc72b3a6-87fc-11e5-8e25-1423f9b7fb42.png)


>  Can you both list the extensions you have installed?

I already listed most of them: https://github.com/EFForg/privacybadgerfirefox/issues/648#issuecomment-154219012
Additionally have NoScript, Blur (formerly DoNotTrackMe), Adblock+ and Greasemonkey (with no scripts for github.com installed) installed, but all other ones really should not be important as they do not interfere with websites.

from an [older reply](https://github.com/EFForg/privacybadgerfirefox/issues/648#issuecomment-153210762:
> subdomains of github.com

Aslo note that most of the domains (the avatar domains at least) are subdomains of githubusercontent.com not github.com.

**Comment by cooperq** [comment]
@rugk are you still seeing github.com blocked by privacy badger anywhere? Can you point me to an exact url?

**Comment by rugk** [comment]
My latest screenshot shows the current state. It seems this URLs are detected but are not detected as tracking me.
(Screenshot from this issue)

**Comment by cooperq** [comment]
okay so github.com is no longer blocked for @rugk how about you @jawz101 are you still having this issue?

**Comment by jawz101** [comment]
After I reinstalled Firefox and the add-on I think I'm good

**Comment by cooperq** [comment]
okay, I'm going to close this for now since no one can reproduce it any more. I will re-open if we can get a reproduction.

**Comment by foxidrive** [comment]
These two threads are closed and seem to focus on github - the problem affects stack overflow and isn't resolved in Firefox.  

https://github.com/EFForg/privacybadgerfirefox/issues/118
https://github.com/EFForg/privacybadgerfirefox/issues/648

The webpage doesn't say what release of Privacy Badger is current but Firefox update settings are default.

**Comment by cooperq** [comment]
@foxidrive can you open another issue and show me some screenshots of the problem, and tell me exactly which domains are being blocked on stack overflow?

**Comment by foxidrive** [comment]
There are no screenshots to show you - when I restart Firefox I have to log into the site again.

I've selected "Click to deactivate Privacy badger on this site" for these two sites
http://stackoverflow.com and http://meta.stackoverflow.com

Bug https://github.com/EFForg/privacybadgerfirefox/issues/118 describes the problem.

**Comment by cooperq** [comment]
Bug #118 is still open so please follow progress and post your info there.
